packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-phone/tests/__snapshots__/gux-form-field-phone.spec.ts.snap

```diff
@@ -93,15 +99,18 @@ exports[`gux-form-field-phone #render label-position should render component as
 <gux-form-field-phone>
   <mock:shadow-root>
     <fieldset class="gux-above gux-form-field-fieldset-container">
-      <legend class="gux-form-field-legend-label gux-screenreader">
-        <slot name="label"></slot>
-        <gux-screen-reader-beta>
-          Required
-        </gux-screen-reader-beta>
-      </legend>
-      <span aria-hidden="true" class="gux-above gux-form-field-legend-label" role="presentation">
-        Label
-      </span>
+      <div class="gux-above gux-form-field-legend-label-container">
+        <legend class="gux-form-field-legend-label gux-screenreader">
+          <slot name="label"></slot>
+          <gux-screen-reader-beta>
+            Required
+          </gux-screen-reader-beta>
+        </legend>
+        <div aria-hidden="true" class="gux-form-field-legend-label" role="presentation">
```

Comment:
I feel like we don't need `role="presentation"` if we have `aria-hidden="true"`

Reply:
I think that's a good question to ask but it would be better if we could kick this to another ticket since this one is already so large. Just I think I'll get mixed up if I change it in the middle of this stuff.